src/index.html

```diff
@@ -1,13 +1,67 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <div 
+        class="card" 
+        data-qa="card"
+      > 
+        <img 
+          class="card__photo"
+          src="images/imac.jpeg" 
+          alt="APPLE A1419 iMac 27"
+        >
+  
+        <h1 class="card__name">
+          APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+  
+        <h2 class="card__code">
+          Product code: 195434
+        </h2>
+        
+        <div class="card__action">
+          <div class="stars stars--1">
+            <div class="stars__star"></div>
+        
+            <div class="stars__star"></div>
+        
+            <div class="stars__star"></div>
+        
+            <div class="stars__star"></div>
+        
+            <div class="stars__star"></div>
+          </div>
```

Comment:
```suggestion
        <div class="card__action">
          <div class="stars stars--1">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```
no need to add blanks between one line elements